kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/KafkaCommandDecoder.java

```diff
@@ -86,7 +86,9 @@ public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) {
 
     protected void close() {
         // Clear the request queue
-        log.info("close channel {} with {} pending responses", ctx.channel(), requestQueue.size());
+        if (log.isDebugEnabled()) {
+            log.debug("close channel {} with {} pending responses", ctx.channel(), requestQueue.size());
+        }
```

Comment:
I think we should still keep the info log level. See https://github.com/streamnative/kop/issues/614#issuecomment-884039479 for my explanation.

Reply:
Yes, when users use log analysis, the info log level may be better.